src/SevenSegmentTM1637.h

```diff
@@ -20,7 +20,13 @@
  #include <WProgram.h>
 #endif
 
-#include <avr/pgmspace.h>   // Used for PROGMEM
+//#include <avr/pgmspace.h>   // Used for PROGMEM
+// esp8266 compatibility
+#if (defined(__AVR__))
+#include <avr\pgmspace.h>
```

Comment:
It's safer to use the forward slash as path separator because the backslash could be seen as an escape character and Windows supports the use of forward slash.